### DISCUSSION TITLE
Can we post the contents of our agenda to github?

### DISCUSSION THREAD

**Discussion by ZIYANGSONG2003** [ZIYANGSONG2003/AI-Enhanced-WordPress-Development-Toolkit#152] [Q&A]
@Austin521

**Comment by 2Jus2** [comment]
I think it would be more suitable for Drive since we've changed this semester documentation to Drive
 @Austin521 Could you upload a new folder and put all the weekly agendas into that folder?

**Reply by Guanweiyi1999** [reply]
Agree
